notebooks/ingestion/dish_ingredients.adb.csv

```diff
@@ -0,0 +1,52 @@
+ConnectionClass,_Image@id,Ingredient@UUID
+HasIngredient,1,f1a2134c-32be-4370-a4e0-411e1ce2f26b
+HasIngredient,1,35ac501b-5ff1-4ff0-9d2f-5794a2f11f44
+HasIngredient,2,4d9931ba-4aee-4846-9078-75704429c982
+HasIngredient,2,8092dcd7-17aa-40e3-ac86-907600b72929
+HasIngredient,3,2f9ac998-ca93-4e28-aa0e-65ca72884e8b
+HasIngredient,3,bb8a84c3-3539-42fb-bf84-06ce459c04d8
+HasIngredient,4,877aec84-4aa1-4a20-b039-0eeb12a25991
+HasIngredient,4,9b74e231-e1a6-4717-8904-117f137cd032
+HasIngredient,4,7d890372-c9fe-4a69-b344-66ab2b8f2d51
+HasIngredient,5,3c6bfd2d-5261-479b-a91c-acaeb261289d
+HasIngredient,5,eef1334c-d36b-40dd-b058-559558278024
+HasIngredient,5,ea02f491-9622-4836-82f8-6f27180103ee
+HasIngredient,6,097cb264-3bce-4ddc-8b54-92c12582d4d0
+HasIngredient,7,e00b45b3-6c66-4987-b3c0-4d46007a0092
+HasIngredient,7,0b7c022a-be58-4629-859f-398d595d6ed8
+HasIngredient,8,d030db31-8eb8-414c-9755-4058b1496068
+HasIngredient,8,46dcac6b-db0c-4142-92e4-518f4495d734
+HasIngredient,8,a07d6e6d-7db3-4839-be5a-090425bd61e4
+HasIngredient,8,ad6f8b2c-0c88-43e8-936b-159da928b255
```

Comment:
Does this schema match with what I added in JSON examples? Let's keep things consistent so they map directly, as much as possible

Reply:
nope. I think you've used the third tab, and I used the second tab for ingredients. Let's clean up the source so that our examples look the same. Also, I think we should make our sheet RO public, so we can better explain to users the origin of the data with a link.